dask/dataframe/shuffle.py

```diff
@@ -495,6 +495,10 @@ def rearrange_by_column(
         if ignore_index:
             df2._meta = df2._meta.reset_index(drop=True)
         return df2
+    elif shuffle == "p2p":
```

Comment:
Suggest this be wrapped in a `try / except` block.  Could make a case for including a warning about the experimental nature.